### PR TITLE
Use compute credits for valgrind job

### PR DESCRIPTION
**Pull request by dergoegge**

### CODE REVIEW

**Comment by dergoegge** [comment]
Timed out at 2 hours, probably needs some kind of permission to use the credits?

File: .cirrus.yml [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+  # Only use credits for pull requests to the qa-assets repo
+  # TODO maybe only use credits if $CIRRUS_BRANCH == 'main'
+  use_compute_credits: $CIRRUS_REPO_FULL_NAME == 'bitcoin-core/qa-assets'
+
 task:
   name: "[system libs, no depends, fuzz, valgrind]  [jammy]"
   container:
     image: ubuntu:22.04
-    cpu: 4
+    cpu: 8
```
[the remark this answers is not in the document]
Comment (dergoegge) [reply .cirrus.yml:14]: I see, thanks! Will update tomorrow.

**Comment by dergoegge** [comment]
Updated config: 3 cores, 12GB, 480m

**Comment by fanquake** [comment]
> lgtm. Maybe just merge to see if it will use credits then?

Ok. We'll merge this now, and see what happens in the CI run.

**Pull request merged**